recipe/meta.yaml

```diff
@@ -8,43 +8,80 @@ package:
 
 source:
   url: https://github.com/ec-jrc/pyPoseidon/archive/{{ version }}.tar.gz
-  sha256: db16f8215bbd534b2c0bce5617894aebdecd76b88016618ba8ce220dca75005e
+  sha256: d2eba8d0710e17b57c7578a4a0433b8e1451b8f19847a36343a0fe57e8b085c7
 
 build:
   noarch: python
   number: {{ build }}
-  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
 
-requirements:
-  build:
-  host:
-    - pip
-    - python >=3.6
-    - setuptools
-    - poetry
-    - poetry-core
-  run:
-    - python >=3.6
-    - mayavi
-    - pyresample
-    - pygeos >=0.7.1
-    - geoviews
-    - rasterio
-    - jigsaw =0.9.12
-    - hvplot
-    - descartes
-    - cfgrib <0.9.9
-    - moviepy
-    - colorlog
-    - f90nml
-    - zarr
+outputs:
+  - name: pyposeidon-base
+    script: build_base.sh
+    build:
+      noarch: python
+    requirements:
+      build:
+        - python                                 # [build_platform != target_platform]
```

Comment:
I really don't think you need this since you are buliding this as a noarch.

Conda-forge build for `linux-64` on `linux-64`, then conda does some magic to make it applicable to other platforms as a python noarch package.

Reply:
I am using as template the recipes from [geopandas](https://github.com/conda-forge/geopandas-feedstock/blob/main/recipe/meta.yaml) and [matplotlib](https://github.com/conda-forge/matplotlib-feedstock/blob/main/recipe/meta.yaml).

Reply:
matplotlib to my knowledge has some compiled features.

geopandas recipe might have a few bugs. One thing that is tricky, is that when a split recipe has the same name as recipe top name, strange things can happen.

Some have started to call the "top" output `pyposeidon-split` and that removes much of that confusion.

Reply:
You can see that in the ci_support files, it doesn't pin python. so that is good

https://github.com/conda-forge/geopandas-feedstock/blob/main/.ci_support/linux_64_.yaml